sprintQuovantis/sprintParam/views.py

```diff
@@ -103,3 +106,76 @@ def post(self, request):
                     "message": "Data not stored"
                 },
                 status=status.HTTP_400_BAD_REQUEST)
+
+
+class UserVote(APIView):
```

Comment:
authentication is missing for all APIs, anyone can make a request without authenticating.